bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/airqHandler.java

```diff
@@ -0,0 +1,956 @@
+                                processType(decObj, "pm2_5", "fineDustConc02_5", "pair");
+                                processType(decObj, "pm10", "fineDustConc10", "pair");
+                                processType(decObj, "pressure", "pressure", "pair");
+                                processType(decObj, "so2", "so2", "pair");
+                                processType(decObj, "sound", "sound", "pair");
+                                processType(decObj, "temperature", "temperature", "pair");
+                                /*
+                                 * We have two places where the Device ID is delivered: with the measurement data and
+                                 * with the configuration.
+                                 * We take the info from the configuration and show it as a property, so we don't need
+                                 * this at this moment. We
+                                 * leave this as a reminder in case for some reason it will be needed in future, e.g.
+                                 * when an air-Q device
+                                 * also sends data from other devices (then with another Device ID)
+                                 *
+                                 * processType(decObj, "DeviceID", "DeviceID", "string");
+                                 */
+                                processType(decObj, "Status", "status", "string");
+                                processType(decObj, "TypPS", "avgFineDustSize", "number");
+                                processType(decObj, "dCO2dt", "dCO2dt", "number");
+                                processType(decObj, "dHdt", "dHdt", "number");
+                                processType(decObj, "door_event", "door_event", "number");
+                                processType(decObj, "health", "health", "number");
+                                processType(decObj, "measuretime", "measuretime", "number");
+                                processType(decObj, "performance", "performance", "number");
+                                processType(decObj, "timestamp", "timestamp", "datetime");
+                                processType(decObj, "uptime", "uptime", "number");
+                                processType(decObj, "tvoc", "tvoc", "pair");
+                            } else {
+                                logger.error("The air-Q data could not be extracted from this string: {}", decEl);
+                            }
+                        }
+                    } catch (Exception e) {
+                        System.out.println("air-Q - airqHandler - polldata.run(): Error while retrieving air-Q data: "
+                                + e.toString());
+                    }
+                }
+            }
+        };
+
+        pollingJob = scheduler.scheduleWithFixedDelay(pollData, 0, POLLING_PERIOD_DATA, TimeUnit.MILLISECONDS);
```

Comment:
```suggestion
        pollingJob = scheduler.scheduleWithFixedDelay(this::pollData, 0, POLLING_PERIOD_DATA, TimeUnit.MILLISECONDS);
```

Consider changing to the following:

Move the code that is in your runnable into a method like below. The "this::pollData" tells it to run the method so you don't need to define a runnable like you did.

public void pollData(){
//code here
}